invoices_app/views.py

```diff
@@ -100,7 +100,9 @@
     THANK_YOU,
     DISCLAIMER,
     INVOICE_STATUS_IN_PROGRESS, INVOICE_STATUS_CHANGES_REQUEST_CODE,
-)
+    INVOICE_STATUS_APPROVED_UPPER, INVOICE_STATUS_APPROVED_EMAIL, INVOICE_STATUS_CHANGES_REQUEST_UPPER,
+    INVOICE_STATUS_CHANGES_REQUEST_EMAIL, INVOICE_STATUS_REJECTED_UPPER, INVOICE_STATUS_REJECTED_EMAIL,
+    INVOICE_STATUS_PAID_UPPER, INVOICE_STATUS_PAID_EMAIL, INVOICE_STATUS_IN_PROGRESS_EMAIL, INVOICE_STATUS_PENDING_CODE)
```

Comment:
put each import in a new line.

Reply:
Take a look of this feature in PyCharm for that https://www.jetbrains.com/help/idea/reformat-file-dialog.html